_posts/2022-09-26-changelog-179.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #179"
+date: 2022-09-26 13:00:00 +0300
+---
+
+This is the last release that supports the 2022.1 IDEs. To receive plugin updates further, please upgrade your IDE to 2022.2.
+
+## New Features
+
+* [#9241] Change the rendering of folded blocks for `use`, `mod`, `extern crate`, and `where` blocks to make them more distinct from folded block comments (by [@afetisov])
+
+  **TODO**: img/gif
+
+* [#9176] Use the [`--keep-going` Cargo option](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#keep-going) when building and evaluating build scripts to continue building even if some crate fails to compile
+
+* [#9325] Enable [build script](https://doc.rust-lang.org/cargo/reference/build-scripts.html) evaluation in nightly plugin builds for 2022.2 platform
+
+* [#9043] Add [`redundant_semicolons`](https://doc.rust-lang.org/rustc/lints/listing/warn-by-default.html#redundant-semicolons) inspection and quick-fix for it
+
+  **TODO**: img/gif
+
+* [#7435] The intention to convert a match expression to a series of if let statements is now offered in more situations (by [@Kobzol])
+
+  **TODO**: img/gif?
+
+* [#9242] update completions for function attributes including `target_feature`, `proc_macro`, `proc_macro_derive`, and `proc_macro_attribute` (by [@afetisov])
+
+## Performance Improvements
+
+* [#9338], [#9337], [#9334], [#9331], [#9322] Slightly speed up name resolution and type inference
+
+* [#9211] Speed up name resolution of large nested paths like `Foo<Foo<Foo<Foo<...>>>>` by resolving and caching them at once
+
+* [#9213] Use dependencies modification tracker in more places to reduce unnecessary cache invalidations
+
+## Fixes
+
+* [#9352] Allow empty precision in formatting parameter (by [@KarelPeeters])
```

Comment:
I think it should be at the end part of "fixes" section since it fixes quite corner case